CONTRIBUTING.md

```diff
@@ -30,8 +30,6 @@ always be in sync.
 
 - `npm start` runs the dev server to run/develop examples
 - `npm test` will run the tests.
-- `scripts/test` same as `npm test` but keeps karma running and watches
```

Comment:
I removed this line because it seems like it's obsolete. When I run `npm test`, I karma keeps running and watches my changes.